torch/_inductor/codegen/wrapper.py

```diff
@@ -1541,7 +1541,10 @@ def prepare_triton_kernel_call(self, device_index, call_args):
         def wrap_arg(arg):
             if isinstance(arg, str):
                 # dynamo wraps unspec variable as 0d CPU tensor, need convert to scalar
-                return arg + ".item()" if V.graph.is_unspec_arg(arg) else arg
+                if V.graph.is_unspec_arg(arg) or V.graph.is_zero_dim_cpu_tensor(arg):
```

Comment:
there are other uses of is_unspec_arg - can we unify the apis ?

Reply:
we can add another api when  `is_unspec_arg ` are both used but we still need them seperately because we have different process for triton kernel signatures. will do in next commit.

Reply:
fixed in https://github.com/pytorch/pytorch/pull/132702/commits/d81b9a48645de323731fb9abfd1e67fbc00ac7e5